site/pages/xprmntl/checkboxes-en.hbs

```diff
@@ -0,0 +1,194 @@
+---
+{
+	"title": "Checkbox and radio button - Provisional feature",
+	"language": "en",
+	"category": "Style",
+	"description": "Working example - provisional feature",
+	"tag": "chckbxrdio",
+	"parentdir": "checkboxes",
+	"altLangPrefix": "checkboxes",
+	"dateModified": "2020-05-11"
+}
+---
+{{>alert-provisional-demos}}
+<span class="wb-prettify all-pre"></span>
+<section class="provisional">
+	<h2>Example 1</h2>
+	<fieldset class="chkbxrdio-grp">
+		<legend>
+		Choose your condiments: <small class="text-muted">(checkboxes + implicit label, large box)</small>
+		</legend>
+		<ul class="list-unstyled lst-spcd-2">
+			<li class="checkbox">
+				<label>
+				<input type="checkbox" id="cond10">
+					<span>Ketchup</span>
+				</label>
+			</li>
+			<li class="checkbox">
+				<label>
+				<input type="checkbox" id="cond11">
+					<span>Relish</span>
+				</label>
+			</li>
+			<li class="checkbox">
+				<label>
+				<input type="checkbox" id="cond12">
+					<span>Mustard</span>
+				</label>
+			</li>
+		</ul>
+	</fieldset>
+	<pre><code>&lt;fieldset class="chkbxrdio-grp"&gt;
+	&lt;legend&gt;Choose your condiments: &lt;small class="text-muted"&gt;(checkboxes + implicit label, large box)&lt;/small&gt;&lt;/legend&gt;
+	&lt;ul class="list-unstyled lst-spcd-2"&gt;
+		&lt;li class="checkbox"&gt;
+			&lt;label&gt;
+			&lt;input type="checkbox" id="cond10"&gt;
+				&lt;span&gt;Ketchup&lt;/span&gt;
+			&lt;/label&gt;
+		&lt;/li&gt;
+		&lt;li class="checkbox"&gt;
+			&lt;label&gt;
+			&lt;input type="checkbox" id="cond11"&gt;
+				&lt;span&gt;Relish&lt;/span&gt;
+			&lt;/label&gt;
+		&lt;/li&gt;
+		&lt;li class="checkbox"&gt;
+			&lt;label&gt;
+			&lt;input type="checkbox" id="cond12"&gt;
+				&lt;span&gt;Mustard&lt;/span&gt;
+			&lt;/label&gt;
+		&lt;/li&gt;
+	&lt;/ul&gt;
+&lt;/fieldset&gt;</code></pre>
+</section>
+<section class="provisional">
+	<h2>Example 2</h2>
+	<fieldset class="chkbxrdio-grp">
+		<legend>
+		Which is your favorite pet: <small class="text-muted">(checkboxes + implicit label, large box)</small>
+		</legend>
+		<ul class="list-unstyled lst-spcd-2">
+			<li class="radio">
+				<label>
+					<input name="cond13" type="radio" id="cond13-a">
+					<span>Dog</span>
```

Comment:
General comment,
we try to not have the need to add accessory "span" only for style. If that is needed, it is preferred to do it via a plugin. In the future plugin might get transform to use shadow dom.
If the new style is only feasible for implicit label and not explicit then make that new style only available for implicit label.